.github/ISSUE_TEMPLATE/30-other-issues.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Other Issues
+about: Use this template for any other non-support related issues
+labels: 'type:others'
+
+---
+
+This template is for miscellaneous issues not covered by the other issue categories.
+
+For questions on how to work with TensorFlow, or support for problems that are not verified bugs in TensorFlow, please go to [StackOverflow](https://stackoverflow.com/tags/tensorflow.js).
```

Comment:
Should be TensorFlow.js, can you verify through out to replace tensorflow with tensorflow.js